release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1058,6 +1081,12 @@ You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8. Yo
 
 * Before this update, if the Cluster Samples Operator encountered an `APIServerConflictError` error, it reported `sample-operator` as having `Degraded` status until it recovered. Momentary errors of this type are not unusual during upgrades and were causing undue concern for administrators monitoring the Operator status. With this update, if the Operator encounters a momentary error, it no longer indicates `openshift-samples` as having `Degraded` status and retries to connect to the API server. Momentary shifts to `Degraded` status should no longer occur. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1993840[*BZ#1993840*])
 
+[discrete]
+[id="ocp-4-10-sro-bug-fixes"]
+==== Special Resource Operators
+
+* Previously, when using `oc explain` for  Special Resource Operator (SRO), the resulting documentation did not provide enough details about the fields. This documentation is now expanded, providing specifics for each field. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2031875[*BZ#2031875*])
```

Comment:
This bug fix and the first fix (BZ#2031875) added in this batch appear to be identical. If this is the case, I believe this description is clearer than the first.